src/app/dashboard/records/record-table.tsx

```diff
@@ -0,0 +1,113 @@
+import { type $Enums } from "@prisma/client";
+import {
+  Badge,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeaderCell,
+  TableRow,
+} from "@tremor/react";
+import { Check, CheckCheck, Loader2, Pencil, Trash2, X } from "lucide-react";
+import { Button } from "~/components/ui/button";
+
+type AttendanceRecord = {
+  attendanceRecordId: string;
+  studentId: string;
+  lectureId: string;
+  status: $Enums.Status;
+  timestamp: Date;
+  studentFullName: string;
+  moduleName: string;
+};
+
+export default function RecordTable({
+  attendanceData,
+  isBeingDeleted,
+  setIsBeingDeleted,
+  deleteRecordById,
+}: {
+  attendanceData: AttendanceRecord[] | undefined;
+  isBeingDeleted: string | null;
+  setIsBeingDeleted: (id: string | null) => void;
+  deleteRecordById: (id: { attendanceRecordId: string }) => void;
+}) {
+  return (
+    <Table className="mt-4">
+      <TableHead>
+        <TableRow>
+          <TableHeaderCell>Attendance Record ID</TableHeaderCell>
+          <TableHeaderCell>Student ID</TableHeaderCell>
+          <TableHeaderCell>Student Name</TableHeaderCell>
+          <TableHeaderCell>Lecture ID</TableHeaderCell>
+          <TableHeaderCell>Module Name</TableHeaderCell>
+          <TableHeaderCell>Status</TableHeaderCell>
+          <TableHeaderCell>Timestamp</TableHeaderCell>
+          <TableHeaderCell>Actions</TableHeaderCell>
+        </TableRow>
+      </TableHead>
+      <TableBody>
+        {attendanceData?.map((record) => (
+          <TableRow key={record.attendanceRecordId}>
+            <TableCell>{record.attendanceRecordId}</TableCell>
+            <TableCell>{record.studentId}</TableCell>
+            <TableCell>{record.studentFullName}</TableCell>
+            <TableCell>{record.lectureId}</TableCell>
+            <TableCell>
+              <Badge color="neutral" size="xs">
+                {record.moduleName}
+              </Badge>
+            </TableCell>
+            <TableCell>
+              {record.status === "PRESENT" && (
+                <Badge color="green" size="sm" icon={CheckCheck}>
+                  {record.status.toLowerCase()}
```

Comment:
```suggestion
                  {record.status.toUpperCase()}
```

Reply:
So this was actually purely for aesthetic reasons that I chose to make it lowercase (cus it's just on frontend rendering side). But I agree, we should switch it to uppercase so it's consistent everywhere.